pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -109,7 +109,7 @@ func (s *volumeStatCalculator) calcAndStoreStats() {
 		if err != nil {
 			// Expected for Volumes that don't support Metrics
 			if !volume.IsNotSupported(err) {
-				klog.V(4).Infof("Failed to calculate volume metrics for pod %s volume %s: %+v", format.Pod(s.pod), name, err)
+				klog.V(4).InfoS("Failed to calculate volume metrics", "pod", format.Pod(s.pod), "volumeName", name, "err", err)
```

Comment:
I don't know what format.Pod does but we should use:
* `klog.KObj` if `s.pod` is an object
* `klog.KRef` otherwise if we have access to pod name and pod namespace

Reply:
format.Pod returns a string of the format `pod.Name, pod.Namespace, pod.UID`
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/util/format/pod.go#L32.

I guess then in this case it is right to go with `klog.KObj`?

Reply:
yes